app.js

```diff
@@ -0,0 +1,41 @@
+import "regenerator-runtime/runtime";
+import axios from "axios";
+
+const BASE = "https://api.github.com/orgs/stackbuilders/repos";
+
+const getAllItems = async () => {
+  try {
+    const response = await axios.get(BASE);
+
+    const allItems = response.data;
+
+    moreThanFive(allItems);
+    lastFiveUp(allItems);
+    sumStars(allItems);
+  } catch (errors) {
+    console.error(errors);
+  }
+};
+getAllItems();
+// const Repos = getAllItems()
+// .then((result) => console.log(result))
+// .catch((err) => {
+// console.error(err);
+// });
+
+const moreThanFive = (Repos) => {
+  console.log(Repos.filter((Repo) => Repo.stargazers_count > 5));
```

Comment:
A good (functional programming) practice is to keep [functions pure](https://medium.com/javascript-scene/master-the-javascript-interview-what-is-a-pure-function-d1c076bec976).
In this case (and others below) you have a side effect -- you are printing to the console.

Probably `moreThanFive` should return the result and let the main function be the one that has all the side effects. What do you think? How do you think purity can help you with (for instance) testing?

Reply:
Yes I think that it is a good idea, I did not realize  the side effects fact, but now I think that it is a good option to return the result and leave those side effects to the main function. Thank you, I already updated it.